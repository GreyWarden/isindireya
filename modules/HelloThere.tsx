import { FC } from 'react'
import styles from '@modules/HelloThere.module.scss'

export const HelloThere: FC = () => (
    <div className={ styles.helloThere__container }>
        Hello there!
    </div>
)
