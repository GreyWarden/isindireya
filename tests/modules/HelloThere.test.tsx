import { render } from "@testing-library/react";
import { HelloThere } from "@modules/HelloThere";

describe('modules/HelloThere.tsx',  () => {
    it('should match snapshot',  () => {
        const {container} = render(<HelloThere />)

        expect(container).toMatchSnapshot()
    });
});