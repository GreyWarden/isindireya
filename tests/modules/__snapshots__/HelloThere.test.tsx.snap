// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`modules/HelloThere.tsx should match snapshot 1`] = `
<div>
  <div>
    Hello there!
  </div>
</div>
`;
